app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::ReviewsController < ApiController
+  def create
+    response = JSON(request.body.read)
+    newReview = response["newReview"]
+    review = Review.new(
+      title: newReview["title"], 
+      description: newReview["description"], 
+      rating: newReview["rating"],
+      animal_id: params[:animal_id],
+      user_id: params["userId"]
+    )
```

Comment:
Please use strong_params to ensure a level of whitelist security and conform with rails convention